assets/js/data/shared-controls.ts

```diff
@@ -188,7 +188,7 @@ export const controls = {
 						if ( errorResponse.body ) {
 							reject( errorResponse.body );
 						} else {
-							reject();
+							reject( errorResponse );
```

Comment:
What did this fix if you don't mind me asking?

Reply:
[This](https://github.com/woocommerce/woocommerce-blocks/pull/5962#pullrequestreview-1037191967) was an issue raised by @opr , the next two comments are related.